book/src/external-health-monitor-controller.md

```diff
@@ -0,0 +1,27 @@
+# CSI external-health-monitor-controller
+
+## Status and Releases
+
+**Git Repository:** [https://github.com/kubernetes-csi/external-health-monitor](https://github.com/kubernetes-csi/external-health-monitor)
+
+**Status:** Alpha 
+
+Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
+--|--|--|--|--|--|--|--
+[external-health-monitor-controller v0.1.0](https://github.com/kubernetes-csi/external-health-monitor/releases/tag/v0.1.0) | [release-0.1](https://github.com/kubernetes-csi/external-health-monitor/tree/release-0.1) | [v1.3.0](https://github.com/container-storage-interface/spec/releases/tag/v1.3.0) | - | k8s.gcr.io/sig-storage/csi-external-health-monitor-controller:v0.1.0 | v1.19 | - | v1.19
```

Comment:
same thing, any k8s version dependencies?

Reply:
It's released after k8s 1.19.

Reply:
Done